examples/ragdoll.js

```diff
@@ -85,7 +85,7 @@ Example.ragdoll = function() {
         counter = 0;
 
     Events.on(engine, 'afterUpdate', function(event) {
-        // tween the timescale for slow-mo
+        // tweak the timescale for slow-mo
```

Comment:
I'm not certain about this

Reply:
I should probably drop this per https://en.wikipedia.org/wiki/Inbetweening (it still doesn't have a page in wikipedia for itself) and even that article isn't particularly comfortable with it, and I think `tween` is being used here as a verb instead of a noun.